File: players.py
class Team(object):
    def __init__(self,name,city,color):
        self.name = name
        self.city = city
        self.color = color
        self.players = []
        
    def display(self):
        for player in self.players:
            print "Team Players:", player.p_name, player.j_num
            return self

class Player(object):
    def __init__(self, p_name, j_num):
        self.p_name = p_name
        self.j_num = j_num

sea = Team("seahawks","seattle","blue")
russ = Player("Russ",3)
sea.players.append(russ)
sea.display()

File: players.js
function Team(name,city,color){
    this.name = name;
    this.city = city;
    this.color = color;
    this.players = [];
}

function Player(p_name,j_number,){
    this.p_name = p_name;
    this.j_number = j_number;
}

var seahawks = new Team("seahawks", "seattle", "blue");

var russ = new Player("r_wilson", 3)
console.log(seahawks)
